images/views.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib import messages
+from django.http import HttpResponseBadRequest, JsonResponse, HttpResponse
+from django.shortcuts import render, redirect
+from django.core.files.images import ImageFile
+from django.core.files.base import ContentFile
+
+from .helpers import image_to_base64str
+from accounts.models import CustomUser
+from hackathon.models import HackTeam, HackProject
+
+VALID_UPLOAD_TYPES = ['profile_image', 'header_image', 'project_image',
+                      'screenshot']
+
+
+def save_image(request):
```

Comment:
Ah, I see this is where the images are saved... base64 🤔 Interesting.

Reply:
Haha yeah, got this from Ray :smile: